code/controllers/subsystem/objects.dm

```diff
@@ -15,6 +15,7 @@ var/datum/subsystem/objects/SSobj
 	NEW_SS_GLOBAL(SSobj)
 
 /datum/subsystem/objects/Initialize(timeofday, zlevel)
+	setupGenetics()
```

Comment:
The objects subsystem that calls the atom movables' initialize() including monkey/initialize() which make the monkey's dna and requires genetics to already be setup. If genetics setup was left in the ticker subsystem (which is initialized after the SSObj) it would fuck up the monkey dna initialization.